src/Features/CSharp/Portable/ReplaceMethodWithProperty/CSharpReplaceMethodWithPropertyService.cs

```diff
@@ -376,11 +376,12 @@ public void ReplaceSetReference(SyntaxEditor editor, SyntaxToken nameToken, stri
                 return;
             }
 
+            var invocation = nameNode?.FirstAncestorOrSelf<InvocationExpressionSyntax>();
+
             var newName = nameChanged
-                ? SyntaxFactory.IdentifierName(SyntaxFactory.Identifier(propertyName).WithTriviaFrom(nameToken))
+                ? SyntaxFactory.IdentifierName(SyntaxFactory.Identifier(propertyName).WithLeadingTrivia(nameToken.LeadingTrivia).WithTrailingTrivia(invocation?.ArgumentList?.CloseParenToken.TrailingTrivia))
```

Comment:
if invocation is null, it feels lke we shoudl explicitly handle that, or else we lose the trailing trivia from nameToken.

Reply:
Added doe to handle condition when invocation is null.